opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -848,13 +753,13 @@ mod tests {
 
     #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
     async fn no_attr_delta_up_down_counter() {
```

Comment:
This test shares lots of duplication with the method below. Can we merge them into one as use table based unit testing here?

Reply:
yes. not fixing too many in this PR, but will do shorter fixes.